src/Boundaries/Sediments/simple_multi_G.jl

```diff
@@ -90,7 +90,7 @@ function SimpleMultiG(grid;
                                            D = -1.3349, 
                                            E = 0.1826, 
                                            F = - 0.0143),
-                      depth = abs(znode(1, grid, Face())),
+                      depth = abs(znodes(grid, Face())[1]),
```

Comment:
`znodes` is user-facing and available as soon as you call `using Oceananigans` so I changed  it here.
this is only called once so it doesn't matter if it's not the most efficient way